extensions/applicationinsights-properties-js/src/Context/Session.ts

```diff
@@ -74,12 +75,14 @@ export class _SessionManager {
             this.initializeAutomaticSession();
         }
 
-        const now = DateTimeUtils.Now();
+        // Always using Date getTime() as there is a bug in older IE instances that causes the performance timings to have the hi-bit set eg 0x800000000 causing
+        // the number to be incorrect.
+        const now = new Date().getTime();
```

Comment:
As per above comment just trying to completely avoid the Hosted IE timing issue (DateTimeUtils.Now() tries to use it), and as this is just a calculation for the cookie expiry time we don't need the extra floating point precision anyway -- so just don't use it.